ui/app/AppLayouts/Chat/components/RenameGroupPopup.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import "../../../../imports"
+import "../../../../shared"
+import "./"
+
+Popup {
+    id: popup
+    modal: true
+
+    Overlay.modal: Rectangle {
+        color: "#60000000"
+    }
+
+    onOpened: {
+        groupName.forceActiveFocus(Qt.MouseFocusReason)
+        groupName.text = chatsModel.activeChannel.name
+    }
+
+    closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
+    parent: Overlay.overlay
+    x: Math.round((parent.width - width) / 2)
+    y: Math.round((parent.height - height) / 2)
+    width: 480
+    height: 159
+    background: Rectangle {
+        color: Theme.white
+        radius: 8
+    }
+    padding: 0
+
+    contentItem: Item {
+        Text {
+            id: groupTitleLabel
+            text: qsTr("Group name")
+            anchors.top: parent.top
+            anchors.left: parent.left
+            font.pixelSize: 13
+            anchors.leftMargin: 16
+            anchors.topMargin: Theme.padding
+            anchors.bottomMargin: Theme.padding
+        }
+
+        Rectangle {
+            id: groupNameBox
+            height: 44
+            color: Theme.grey
+            anchors.top: groupTitleLabel.bottom
+            radius: 8
+            anchors.right: parent.right
+            anchors.left: parent.left
+            anchors.leftMargin: Theme.padding
+            anchors.rightMargin: Theme.padding
+            anchors.topMargin: 7
+
+            TextField {
+                id: groupName
+                placeholderText: qsTr("Group Name")
+                anchors.left: parent.left
+                anchors.right: parent.right
+                anchors.verticalCenter: parent.verticalCenter
+                font.pixelSize: 15
+                background: Rectangle {
+                    color: "#00000000"
+                }
+                width: groupNameBox.width - 32
+                selectByMouse: true
+            }
+        }
+
+        StyledButton {
+            id: saveBtn
+            anchors.top: groupNameBox.bottom
+            anchors.topMargin: 22
+            anchors.right: parent.right
+            anchors.rightMargin: Theme.padding
+            label: qsTr("Save")
+            onClicked : {
+                chatsModel.renameGroup(groupName.text)
+                popup.close();
+            }
+        }
+  }
+}
```

Comment:
Any reason this one couldn't be done using `ModalPopup`?

Reply:
![image](https://user-images.githubusercontent.com/1106587/84771053-c0a9b200-afa6-11ea-9c27-1474b59262cc.png)
The design does not use a header, and has a different height, no close button and no separators. 

To use the popup modal I'd have to update it to support these features for something that at the moment seems to happen only for a specific scenario